### ISSUE TITLE
Bug: Add Aria-Live label to filter count on CDD 

### ISSUE THREAD

**Issue by MaeganWood64** [US-EPA-CAMD/PI-1#4434]
## Requirements

As per Brian on 10/28 "  see only one issue in the filter changes:
6B – 4.1.2 - change-notify-links - When the count is added above the left column over the Filters, this should be encapsulated in an aria-live region so that the number is conveyed as it refreshes."

**Comment by aprematta** [comment]
Looks good, aria live assertive has been introduced
